packages/client-sdk-nodejs/src/config/transport/transport-strategy.ts

```diff
@@ -156,6 +167,21 @@ export class StaticGrpcConfiguration implements GrpcConfiguration {
       numClients: numClients,
     });
   }
+
+  getConcurrentRequestsLimit(): number {
+    return this.concurrentRequestsLimit;
+  }
+
+  withConcurrentRequestsLimit(
+    concurrentRequestsLimit: number
+  ): GrpcConfiguration {
+    return new StaticGrpcConfiguration({
+      deadlineMillis: this.deadlineMillis,
+      maxSessionMemoryMb: this.maxSessionMemoryMb,
+      numClients: this.numClients,
+      concurrentRequestsLimit: concurrentRequestsLimit,
+    });
```

Comment:
As a follow up PR we should validate the concurrentRequestsLimit is strictly positive.